tests/core/engine_adapter/integration/test_integration.py

```diff
@@ -750,7 +750,7 @@ def test_insert_overwrite_by_time_partition(ctx: TestContext):
     results = ctx.get_metadata_results()
     assert len(results.views) == 0
     assert len(results.materialized_views) == 0
-    assert len(results.tables) == len(results.non_temp_tables) == 1
+    assert len(results.tables) == 2 if ctx.dialect == "clickhouse" else 1
```

Comment:
What's the extra table produced by Clickhouse that the other adapters dont have?